products/green_fast_track/models/intermediate/flags/int_flags__spatial.sql

```diff
@@ -75,3 +75,4 @@ SELECT
         ELSE ST_DISTANCE(bbl_geom, raw_geom)
     END AS distance
 FROM joined_hexes
+ORDER BY bbl ASC, variable_type ASC, variable_id ASC
```

Comment:
I might keep this ordering in the aggregate in `green_fast_track_bbls` - postgres makes no guarantee of ordering of results returned by a query, and depending on how the compiler is feeling in terms of optimizing that aggregation, I think it makes sense to explicitly order where we want things ordered

Reply:
see [be14145](https://github.com/NYCPlanning/data-engineering/pull/826/commits/be14145e946325b8f42ac056b9a6553687cb3c46). outputs didn't change (a good thing) but I definitely like the extra confidence in the ordering